xdsl/interactive/passes.py

```diff
@@ -26,7 +26,7 @@ def get_new_registered_context(
     """
     Generates a new MLContext, registers it and returns it.
     """
-    ctx = MLContext(True)
+    ctx = MLContext()
```

Comment:
If we want to go ahead with this change then could we at least have allow_unregistered_dialect passed in as a command line argument?